### PR TITLE
[React, SDK] New EmbeddedWallet API

### PR DESCRIPTION
**Pull request by ElasticBottle**
## Problem solved

Exposes recovery codes in `wallets/embedded-wallet` return type. To enable user managed flow for embedded wallets

## Changes made

- [ ] Public API changes: `sendEmailOtp` in `EmbeddedWallet` to easily initiate the headless email otp flow
- [ ] Internal API changes: `connect` in `EmbeddedWallet` now takes in an optional `recovoeryCode` param for `headless_email_otp_verification` login type

## How to test

- [ ] Automated tests: NONE
- [ ] Manual tests: Test out in test app

### CODE REVIEW

File: .changeset/tame-buckets-serve.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+"@thirdweb-dev/wallets": minor
+"@thirdweb-dev/react": minor
+---
+
+feat (wallets): Add support for user managed recovery code login flows for embedded wallets in all flows except Google Login
```
Comment (edwardysun) [inline .changeset/tame-buckets-serve.md:6]: User-managed encryption key (instead of recovery code)

File: .changeset/rich-llamas-do.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  },
+});
+
+// disable google sign in
+paperWallet({
+  oauthOptions: false,
+});
+
+// Using USER_MANAGED recovery also disables google sign in
+paperWallet({
```
Comment (iketw) [inline .changeset/rich-llamas-do.md:43]: Are we missing:

```
embeddedWallet({
  advancedOptions: {
    recoveryShareManagement: "USER_MANAGED",
  },
});
```

**Pull request closed without merging**